sdk/core/azure-core/test/libcurl-stress-test/libcurl_stress_test.cpp

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @brief Validates the Azure Core transport adapters with fault responses from server.
+ *
+ * @note This test requires the Http-fault-injector
+ * (https://github.com/Azure/azure-sdk-tools/tree/main/tools/http-fault-injector) running. Follow
+ * the instructions to install and run the server before running this test.
+ *
+ */
+#define REQUESTS 100
+#define WARMUP 100
+#define ROUNDS 100
+
+#include <azure/core.hpp>
+#include <azure/core/http/curl_transport.hpp>
+#include <fstream>
+#include <ios>
+#include <iostream>
+#include <memory>
+#include <stdlib.h>
+#include <string>
+#include <vector>
+
+void SendRequest(std::string target)
+{
+  std::cout << target << std::endl;
+  /* The transport adapter must allow insecure SSL certs.
+  If both curl and winHttp are available, curl is preferred for this test.for*/
+#if defined(BUILD_CURL_HTTP_TRANSPORT_ADAPTER)
+  Azure::Core::Http::CurlTransportOptions curlOptions;
+  curlOptions.SslVerifyPeer = false;
+  auto implementationClient = std::make_shared<Azure::Core::Http::CurlTransport>(curlOptions);
+
+#elif (BUILD_TRANSPORT_WINHTTP_ADAPTER)
+  Azure::Core::Http::WinHttpTransportOptions winHttpOptions;
+  auto implementationClient = std::make_shared<Azure::Core::Http::WinHttpTransport>(winHttpOptions);
+#endif
+  try
+  {
+
+    Azure::Core::Context context;
+    //  auto duration = std::chrono::milliseconds(1000);
+    // auto deadline = std::chrono::system_clock::now() + duration;
+    auto request
+        = Azure::Core::Http::Request(Azure::Core::Http::HttpMethod::Get, Azure::Core::Url(target));
+    auto response = implementationClient->Send(request, context); //.WithDeadline(deadline));
+    // Make sure to pull all bytes from network.
+    auto body = response->ExtractBodyStream()->ReadToEnd();
+  }
+  catch (std::exception const&)
+  {
```

Comment:
Should we log or print out the exceptions, rather than silently ignore them?
If we want to ignore, by design, please add a comment.